src/test/java/com/intuit/graphql/orchestrator/integration/FieldResolverDirectiveToplevelTest.java

```diff
@@ -36,7 +36,7 @@
 
   private static final String USERS_DOWNSTREAM_QUERY = "query GetQuery {users {id firstName lastName}}";
 
-  private static final String BOOKS_SIMPLE_DOWNSTREAM_QUERY = "query GetQuery {books {id name author {}}}";
+  private static final String BOOKS_SIMPLE_DOWNSTREAM_QUERY = "query GetQuery {books {id name author {petId}}}";
```

Comment:
petId is a required field and will be automatically added to downstream query if not selected.